app/util/analytics.py

```diff
@@ -130,37 +124,47 @@ def get_actual_test_count(self):
             elif selenium_test in line:
                 self.selenium_test_count = self.selenium_test_count + 1
 
+    def __convert_to_sec(self, time):
+        seconds_per_unit = {"s": 1, "m": 60, "h": 3600, "d": 86400, "w": 604800}
+        time = str(time)
+        numbers = ''.join(filter(str.isdigit, time))
+        units = ''.join(filter(str.isalpha, time))
+        return int(numbers) * seconds_per_unit[units] if units in seconds_per_unit else int(numbers)
+
+
     def generate_analytics(self):
         self.application_url = self.config_yml.server_url
-        self.run_id = self.id_generator(string=self.application_url)
         self.concurrency = self.config_yml.concurrency
-        self.duration = self.config_yml.duration
+        self.duration = self.__convert_to_sec(self.config_yml.duration)
         self.os = self.get_os()
         self.actual_duration = self.get_actual_run_time()
         self.tool_version = TOOLKIT_VERSION
         self.get_actual_test_count()
-        self.date = datetime.utcnow().strftime("%m/%d/%Y-%H:%M:%S")
+
 
 class AnalyticsSender:
 
     def __init__(self, analytics_instance):
-        self.run_analytics = analytics_instance
+        self.analytics = analytics_instance
 
     def send_request(self):
-        base_url = BASE_URL
-        params_string = f'date={self.run_analytics.date}&' \
-                        f'app_type={self.run_analytics.application_type}&' \
-                        f'os={self.run_analytics.os}&' \
-                        f'tool_ver={self.run_analytics.tool_version}&' \
-                        f'run_id={self.run_analytics.run_id}&' \
-                        f'exp_dur={self.run_analytics.duration}&' \
-                        f'act_dur={self.run_analytics.actual_duration}&' \
-                        f'sel_count={self.run_analytics.selenium_test_count}&' \
-                        f'jm_count={self.run_analytics.jmeter_test_count}&' \
-                        f'concurrency={self.run_analytics.concurrency}'
-
-        r = requests.get(url=f'{base_url}{params_string}')
-        if r.status_code != 403:
+        headers = {"Content-Type": "application/json"}
+        params = {"date": self.analytics.date,
+                  "time_stamp": datetime.now().timestamp(),
+                  "app_type": self.analytics.application_type,
+                  "os": self.analytics.os,
+                  "tool_ver": self.analytics.tool_version,
+                  "run_id": self.analytics.run_id,
+                  "exp_dur": self.analytics.duration,
+                  "act_dur":  self.analytics.actual_duration,
+                  "sel_count": self.analytics.selenium_test_count,
+                  "jm_count": self.analytics.jmeter_test_count,
+                  "concurrency": self.analytics.concurrency
+        }
+        r = requests.post(url=f'{BASE_URL}', json=params, headers=headers)
+        response = r.json()
```

Comment:
can confusing, let's name it `response_body`

Reply:
`response` and `r` are the typical name usage for this case in Python (according to multiple documentation and examples). Do not think that it might be confusable.

Reply:
I randomly looked at the python code on the internet:
https://www.geeksforgeeks.org/get-post-requests-using-python/
https://realpython.com/python-requests/
https://python-scripts.com/requests

In some cases JSON as `data` in other `json_response` but not found `response` usage for JSON data. But `response` name is used as the name for Http response in some cases but in some cases as `r`. I am ok with `r`.
But using variable name: `response` for JSON in the scope where you have real Http response is confusing. A real response consists not only from JSON data, but it also has other data, also some people using `response` instead of `r` so I think it is very confusing to use `response` for JSON data.